src/main/java/com/pubnub/api/endpoints/objects_vsp/space/CreateSpace.java

```diff
@@ -0,0 +1,45 @@
+package com.pubnub.api.endpoints.objects_vsp.space;
+
+import com.pubnub.api.PubNub;
+import com.pubnub.api.endpoints.objects_api.CompositeParameterEnricher;
+import com.pubnub.api.endpoints.objects_api.utils.Include.CustomIncludeAware;
+import com.pubnub.api.endpoints.objects_vsp.SpaceEndpoint;
+import com.pubnub.api.managers.RetrofitManager;
+import com.pubnub.api.managers.TelemetryManager;
+import com.pubnub.api.managers.token_manager.TokenManager;
+import com.pubnub.api.models.consumer.objects_vsp.space.Space;
+import com.pubnub.api.models.consumer.objects_vsp.space.CreateSpaceResult;
+import com.pubnub.api.models.server.objects_api.EntityEnvelope;
+
+import java.util.Map;
+
+public abstract class CreateSpace extends SpaceEndpoint<CreateSpace, EntityEnvelope<Space>, CreateSpaceResult> implements CustomIncludeAware<CreateSpace> {
```

Comment:
Why the return value is `CreateSpaceResult` and not just `Space`? In Kotlin we return just `Space`. Could you please align the return values for all new methods to what we have in Kotlin already?

Reply:
I forgot that in Kotlin we get rid of status in response. I will change `CreateSpaceResult` to `Space`. Status might be inferred by Space object being present or exception being thrown
e.g. when creating space with id that already exists following exception is thrown:
{"error":{"message":"Requested resource already exists.","source":"metadata"},"status":409}